spec/rails_helper.rb

```diff
@@ -6,14 +6,6 @@
 require 'spec_helper'
 require 'rspec/rails'
 require 'capybara/rails'
-
-Shoulda::Matchers.configure do |config|
-  config.integrate do |with|
-    with.test_framework :rspec
-    with.library :rails
-  end
-end
```

Comment:
@concach Do you prefer not to use shoulda-matchers? With shoulda-matchers, our model specs would be much shorter & cleaner (see my comment on spec/models/invoice_spec.rb)